src/main/java/io/strimzi/kafka/bridge/mqtt/config/KafkaConfig.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.mqtt.config;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Represents configurations related to Kafka
+ * @see AbstractConfig
+ */
+public class KafkaConfig extends AbstractConfig {
+
+    // Prefix for all the specific configuration parameters for Kafka in the properties file
+    public static final String KAFKA_CONFIG_PREFIX = "kafka.";
+
+    // Kafka bootstrap server configuration. This will be removed when we add the apache kafka client
+    public static final String BOOTSTRAP_SERVERS_CONFIG = KAFKA_CONFIG_PREFIX + "bootstrap.servers";
+
+    private final KafkaProducerConfig kafkaProducerConfig;
+    private final KafkaAdminConfig kafkaAdminConfig;
```

Comment:
Kafka admin? I guess this is coming from the Strimzi HTTP bridge, which is fine but we should remove everything we don't need like this Kafka admin related config.

Reply:
Sure!